### PR TITLE
docs: nodejs trademark policy link broken

### PR DESCRIPTION
**Pull request by kghamilton89**
docs: nodejs trademark policy link broken, this pr updates to valid value

Notes: none

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,4 +112,4 @@ and more can be found on the [Community page](https://www.electronjs.org/communi
 
 [MIT](https://github.com/electron/electron/blob/main/LICENSE)
 
-When using Electron logos, make sure to follow [OpenJS Foundation Trademark Policy](https://openjsf.org/wp-content/uploads/sites/84/2021/01/OpenJS-Foundation-Trademark-Policy-2021-01-12.docx.pdf).
+When using Electron logos, make sure to follow [OpenJS Foundation Trademark Policy](https://images.prismic.io/openjsf/69641abf-ac84-44a2-b719-ee4498d07b06_OpenJS-Foundation-Trademark-Policy-2021-01-12.docx.pdf).
```
Comment (codebytere) [inline README.md:115]: We should have this go to https://trademark-policy.openjsf.org/ - it routes to the link you changed it to but is more robust in allowing for future changes.

**Review by codebytere — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kghamilton89) [reply README.md]: @codebytere leaving conversation open for maintainers if needed, but this has been actioned ⚡

**Review by codebytere — APPROVED** [review]
thanks!

**Review by jkleinsc — APPROVED** [review]

**Comment by jkleinsc** [comment]
Merging as CI failure unrelated to PR change.

**Pull request merged**